examples/README.md

```diff
@@ -0,0 +1,185 @@
+# Examples
+
+RIOT provides a wide range of examples to demonstrate the capabilities of the OS and its modules.
+
+The examples are organized in subdirectories of the `examples` directory. Each example directory contains a `README.md` file that provides information about the example and how to run it.
+
+Here is a quick overview of the examples available in the RIOT repository:
+
+### The Essentials
+
+- [default](./default)
+  - This application is a showcase for RIOT's hardware support. Using it
+    for your board, you should be able to interactively use any hardware
+    that is supported.
+- [hello-world](./hello-world)
+  - A simple "Hello World" that shows the basic structure of a RIOT application.
+- [blinky](./blinky)
+  - The classic "Hello World" example for embedded systems: blinking an LED (or printing "Blink!" via stdio when none are available).
+- [leds_shell](./leds_shell)
+  - The application `leds_shell` is a basic example, which allows easy, interactive
+    control of internal board LEDs, and basic GPIO for externally connected simple
+    devices (for e.g. additional LEDs, relay, motors - via dedicated drivers, etc.)
+    via the shell.
+- [saul](./saul)
+  - This example demonstrates the usage of the SAUL (Sensor Abstraction Layer User) module.
```

Comment:
That's not SAUL ;-) https://doc.riot-os.org/group__drivers__saul.html

Reply:
SALU 😄 Good catch